security-features/privilege-restriction/index.rst

```diff
@@ -53,6 +53,46 @@ Regression tests: `test-kernel-security.py <https://git.launchpad.net/qa-regress
 SMACK
 =====
 
-SMACK is a flexible inode-based MAC.
+SMACK (Simplified Mandatory Access Control Kernel) is a LSM that implements a label-based Mandatory Access Control (MAC) framework designed for simplicity and minimal administrative overhead. Unlike complex policy-driven systems, SMACK uses straightforward text labels assigned to subjects (processes) and objects (files, sockets, etc.) with access decisions made through simple rule comparisons. This architecture makes SMACK particularly well-suited for embedded systems, IoT devices and environments where security policies need to be easily understood and maintained without extensive expertise.
 
-Regression tests: `test-kernel-security.py <https://git.launchpad.net/qa-regression-testing/tree/scripts/test-kernel-security.py>`_.
+Access rules are expressed as::
+
+    <subject-label> <object-label> <access>
+
+where access describes the access permitted using the traditional Linux read (R or r) access, write (W or w) access, execute (X or x) access, or append (A or a) access. A dash (-) can be used as a place holder or to express that no access be permitted if used by itself.
+
+SMACK uses several special system labels:
+
+- ``_`` pronounced "floor"
+- ``*`` pronounced "star"
+- ``^`` pronounced "hat"
+
+There are a limited number of pre-defined rules:
+
++---------------+---------------+--------+
+| Subject Label | Object Label  | Access |
++===============+===============+========+
+| \*            | any           | \-     |
++---------------+---------------+--------+
+| any           | \*            | rwxa   |
++---------------+---------------+--------+
+| ordinary      | ordinary      | rwxa   |
++---------------+---------------+--------+
+| any           | \_            | rx     |
++---------------+---------------+--------+
+| ^             | any           | rx     |
++---------------+---------------+--------+
+
+The third rule uses "ordinary" to refer to any label except ``*`` and describes the case where the subject label and the object label are the same.
```

Comment:
It might be useful to express a quick example here to really reinforce this is only true for labels that are *exactly* the same. Maybe something like: 
```suggestion
The third rule uses "ordinary" to refer to any label except ``*`` and describes the case where the subject label and the object label are the same. For example, a subject "UserApp" accessing an object "UserApp" would be granted read, write, execute, and append access due to the third rule, while the same subject "UserApp" accessing an object "DatabaseApp" would not be granted that access (unless another rule exists that allows such access).
```